From 08e9a71c05c61d362f60d4c558cfe1a2109f1c3d Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Wed, 25 Jul 2018 10:11:41 +0100
Subject: [PATCH 25/26] config/common: Clean up debug packages

 * Include all debug packages via PRODUCT_PACKAGES_DEBUG.
   This whole list can likely be removed during P bringup,
   considering the current master branch status.

 * Remove oprofiled and sqlite3 from the list, because
   the former no longer exists and the latter is already
   part of build/target/product/core_base.mk.

Change-Id: Icde54061cd5a85d381b756364287344c3d216957
---
 config/common.mk | 12 +++++-------
 1 file changed, 5 insertions(+), 7 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index e92e66c5..c447c520 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -179,12 +179,8 @@ PRODUCT_PACKAGES += \
     gdbserver \
     htop \
     libsepol \
-    micro_bench \
-    oprofiled \
     pigz \
     powertop \
-    sqlite3 \
-    strace \
     vim \
     wget
 
@@ -232,11 +228,13 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     media.recorder.show_manufacturer_and_model=true
 
 # These packages are excluded from user builds
-ifneq ($(TARGET_BUILD_VARIANT),user)
-PRODUCT_PACKAGES += \
+PRODUCT_PACKAGES_DEBUG := \
+    micro_bench \
     procmem \
-    procrank
+    procrank \
+    strace
 
+ifneq ($(TARGET_BUILD_VARIANT),user)
 # Conditionally build in su
 ifeq ($(WITH_SU),true)
 PRODUCT_PACKAGES += \
-- 
2.17.1

